jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -361,6 +361,8 @@ public void invoke(final long window, final int width, final int height) {
         });
 
         allowSwapBuffers = settings.isSwapBuffers();
+        oldFramebufferWidth = settings.getWidth();
+        oldFramebufferHeight = settings.getHeight();
```

Comment:
As I mentioned at #1945, I'm not convinced this is the right thing to do.

I'd rather omit this delta and leave `oldFrameBufferHeight` and `oldFrameBufferWidth` at their initial values of `0`.

Reply:
what about initializing them to the width and height values read from `glfwGetFramebufferSize(window, width, height);`?

Reply:
That would be better than using `settings`, but I still prefer initializing to zero.

That way, if the app registers a `SystemListener` early enough, it will get a `reshape()` callback after the window is created. That could be useful. It's the ideal time to lay out the GUI, for instance.

Reply:
I see, then let's close this PR and I will submit a new one for the`if (requestWidth <= 0 || requestWidth <= 0)`.

Thank you so much for reviewing